From 68e0304552c8e7df17e599a69b6f4b735844f95b Mon Sep 17 00:00:00 2001
From: Sumit Singh <sumsingh@nvidia.com>
Date: Wed, 23 Apr 2014 11:06:58 +0530
Subject: [PATCH 19/27] asm-generic: processor.h: remove redundant macros

Removing cpu_relaxed_read and cpu_relaxed_read_long macros from
processor.h, as these macros are defined in asm-generic/relaxed.h.

Bug 1440421

Change-Id: I5d1ba25755e1c9d33b080dfe01ba838289f306af
Signed-off-by: Sumit Singh <sumsingh@nvidia.com>
Reviewed-on: http://git-master/r/400093
(cherry picked from commit 57eb21e2d4cad3ce1f85283cfffd0eff85a6d17d)
Reviewed-on: http://git-master/r/422209
Reviewed-by: Automatic_Commit_Validation_User
GVS: Gerrit_Virtual_Submit
Reviewed-by: Bharat Nihalani <bnihalani@nvidia.com>
Signed-off-by: Jesse Chan <jc@linux.com>
Signed-off-by: Michele Beccalossi <beccalossi.michele@gmail.com>
---
 include/asm-generic/processor.h | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/include/asm-generic/processor.h b/include/asm-generic/processor.h
index cba2d9edf73..0b57d920cdd 100644
--- a/include/asm-generic/processor.h
+++ b/include/asm-generic/processor.h
@@ -15,13 +15,7 @@
 #ifndef _ASM_GENERIC_PROCESSOR_H_
 #define _ASM_GENERIC_PROCESSOR_H_
 
-#ifndef cpu_relaxed_read
-#define cpu_relaxed_read(p) (*(p))
-#endif
-
-#ifndef cpu_relaxed_read_long
-#define cpu_relaxed_read_long(p) (*(p))
-#endif
+#include <asm-generic/relaxed.h>
 
 #ifndef cpu_read_relax
 #define cpu_read_relax() cpu_relax()
-- 
2.11.0

